lib/emque/producing/message/message.rb

```diff
@@ -36,6 +36,14 @@ def read_raise_on_failure
           end
         end
 
+        def ignored_exceptions(ignored_exceptions)
+          @ignored_exceptions = Array(ignored_exceptions) + [InvalidMessageError, MessagesNotSentError]
```

Comment:
I think the default behavior should be to raise as opposed to ignore.
suggested revision:

```ruby
def ignore_exception(*exception_classes)
  @ignored_exceptions = (read_ignored_expections + exception_classes).uniq
end

def read_ignored_exceptions
  (@ignored_exceptions + Emque::Producing.configuration.ignored_exceptions).uniq
end
```

then the rescue becomes

```ruby
rescue *ignored_exceptions => error
```